packages/trader/src/Modules/Trading/Components/Form/TradeParams/allow-equals.tsx

```diff
@@ -17,21 +33,23 @@ const AllowEquals = ({
     onChange,
     value,
     has_equals_only,
-}) => {
+}: TAllowEquals) => {
     const has_callputequal_duration = hasDurationForCallPutEqual(
         contract_types_list,
-        duration_unit,
+        duration_unit as TDurationUnit,
```

Comment:
```suggestion
        duration_unit,
```
@henry-deriv it should work without it since its type in allow-equals.ts is TTradeStore['duration_unit']

Reply:
It works in 4th package and above, can be removed from here